README.md

```diff
@@ -1,3 +1,19 @@
+<!-- 
+  Hivemall: Hive scalable Machine Learning Library
```

Comment:
Need this title line?
I checked license headers in other apache projects and found all the same format they have;
ex.)
https://github.com/apache/incubator-systemml/blob/master/scripts/sparkDML.sh
https://github.com/apache/bahir/blob/master/bin/run-example
https://github.com/apache/flink/blob/master/tools/create_release_files.sh
...